src/DICOM.jl

```diff
@@ -398,7 +398,7 @@ function element(st::IOStream, evr::Bool, dcm=emptyDcmDict, dVR=Dict{Tuple{UInt1
 
     vr == "AT" ? [ read!(st, Array{UInt16}(undef, 2)) for n=1:div(sz,4) ] :
 
-    vr == "AS" ? String(read!(st, Array{UInt8}(undef, 4))) :
+    vr == "AS" ? String(read!(st, Array{UInt8}(undef, sz))) :
```

Comment:
Looks like this can be merged into line #378?

Reply:
Done! Good catch.